guides/v2.1/cloud/env/variables-deploy.md

```diff
@@ -424,15 +424,29 @@ stage:
 -  **Default**—`true`
 -  **Version**—Magento 2.1.4 and later
 
-On deployment, replace Magento base URLs in the database with project URLs. This is useful for local development, where base URLs are set up for your local environment. When you deploy to a Cloud environment, we change the URLs so you can access your storefront and Magento Admin using project URLs.
+On deployment, replace Magento base URLs in the database the project URLs specified in the `MAGENTO_CLOUD_ROUTES` variable. This is useful for local development, where base URLs are set up for your local environment. 
+When you deploy to a Cloud environment, we change the URLs so you can access your storefront and Magento Admin using project URLs.
+
+To update URLs when deploying to staging or production environments,                             use the `FORCE_UPDATE_URLS` variable. 
 
 ```yaml
 stage:
   deploy:
     UPDATE_URLS: false
 ```
 
-You should set this variable to `false` _only_ in Staging or Production environments, where the base URLs cannot change. For Pro, we already set this to `false` for you.
+### `FORCE_UPDATE_URLS`
+
+-  **Default**—`true`
+-  **Version**—Magento 2.1.4 and later
+
+On deployment to staging and production environments, this variable replaces Magento base URLs in the database with the project URLs specified in the `MAGENTO_CLOUD_ROUTES` variable. Use this setting to override the default behavior of the UPDATE_URLS variable which is ignored when deploying to staging or production environments. 
```

Comment:
Link to variable definition? Just a thought, not a requirement. Maybe it's not necessary.
Caps for the environments.

Reply:
so i intentionally didn't cap to indicate both Pro and Starter staging and production.   Initially, I had listed out the environments: Pro Production and Staging, and Starter Production (`master` branch) and Staging.    I was on the fence about this.

Reply:
Ahhhhh, so just generalizing... 👍